compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ReachabilityAnalyzer.java

```diff
@@ -728,8 +728,8 @@ public void visit(BLangDoClause doClause, AnalyzerData data) {
     }
 
     @Override
-    public void visit(BLangWorkerSend workerSendNode, AnalyzerData data) {
-        checkStatementExecutionValidity(workerSendNode, data);
+    public void visit(BLangWorkerAsyncSendExpr asyncSendExpr, AnalyzerData data) {
+        checkStatementExecutionValidity(asyncSendExpr, data);
```

Comment:
Since `BLangWorkerAsyncSendExpr` is a `BLangExpression` now, we don't need to check statement validity for `asynSendExpr`. Therefore we can get rid of calling this function here.

Reply:
Updated 40cc512e09d1ce6989618c1c7392ab833ab772b5